src/main/resources/com/xebialabs/xlt/ci/XLTestView/config.jelly

```diff
@@ -32,6 +32,9 @@
                     <f:entry field="excludes" title="${%Exclude pattern}">
                         <f:textbox default=""/>
                     </f:entry>
+                    <f:entry title="${%XL TestView can make build unstable}">
+                        <f:checkbox checked="true" field="makeUnstable" />
```

Comment:
Also add a help-makeUnstable under TestSpecificationDescribable with a help message see help-excludes.html for instance.